lib/PromptSession.ts

```diff
@@ -265,3 +265,20 @@ export class PromptSession {
 		return answers;
 	}
 }
+
+export function chooseTerm() {
+	return inquirer.prompt({
```

Comment:
you can use `await` to cut down on promise handlers :)
`const answers = await inquirer.prompt(...```